beanquery/shell.py

```diff
@@ -777,14 +777,15 @@ def create_custom_query_map(entries):
 @click.argument('query', nargs=-1)
 @click.option('--numberify', '-m', is_flag=True,
               help="Numberify the output, removing the currencies.")
-@click.option('--format', '-f', type=click.Choice(_SUPPORTED_FORMATS),
+@click.option('--format', '-f', 'output_format',
+              type=click.Choice(_SUPPORTED_FORMATS),
               default=_SUPPORTED_FORMATS[0], help="Output format.")
 @click.option('--output', '-o', type=click.File('w'), default='-',
               help="Output filename.")
 @click.option('--no-errors', '-q', is_flag=True,
               help="Do not report errors.")
 @click.version_option(message=VERSION)
-def main(filename, query, numberify, format, output, no_errors):
+def main(filename, query, numberify, output_format, output, no_errors):
```

Comment:
Redefining built-in 'format' (redefined-builtin)

Reply:
Another IMHO useless pylint check.